### ISSUE TITLE
Add an emoji picker to the calendar event summary

### ISSUE THREAD

**Issue by miaulalala** [nextcloud/calendar#4716]
### Is your feature request related to a problem? Please describe.

Please see my mediocre mockup on how it could look:

![image](https://user-images.githubusercontent.com/7427347/199954714-546149ce-b5f0-479c-bb74-05746287a11c.png)

### Additional context

I have tested if emojis are written to the DB correctly and they are. Export contains the emoji as well:

```ìcs
BEGIN:VEVENT
CREATED:20221103T164654Z
DTSTAMP:20221104T103633Z
LAST-MODIFIED:20221104T103633Z
SEQUENCE:4
UID:e6d9730d-2f5f-4efe-9e54-bf00daa0258d
DTSTART;TZID=Europe/Vienna:20221106T090000
DTEND;TZID=Europe/Vienna:20221106T140000
STATUS:CONFIRMED
SUMMARY:🏖️  Beach Luau (Bring something special for the potluck!)
END:VEVENT
```

**Comment by tcitworld** [comment]
It would be nice to test in common 3rd-party clients if they correctly show up emoji, and how they behave if we use recently introduced emojis that they might not have support for.

**Comment by miaulalala** [comment]
Google Calendar (web):

![gcal](https://user-images.githubusercontent.com/7427347/202689773-a77d4f95-fb83-4ea6-b685-862195d896b9.png)

Google Client (web) when the emoji isn't available:

![image](https://user-images.githubusercontent.com/7427347/202694723-5484590e-d0aa-43ec-9afd-2c5396a9c298.png)

Android integrated Calendar app with Google Calendar:

![signal-2022-11-18-115946_002](https://user-images.githubusercontent.com/7427347/202690366-a60ece0b-8e3d-43f5-b292-facb0f9fce55.jpeg)

Thunderbird 102:

![image](https://user-images.githubusercontent.com/7427347/202690555-7dceba73-2a31-4361-8a0a-8ba6f1105d0d.png)
![image](https://user-images.githubusercontent.com/7427347/202690618-8ed2c698-9a91-449a-a105-b8171b1caef3.png)

GMX (1&1 webmail) web calendar:

![gmx](https://user-images.githubusercontent.com/7427347/202691026-6ea5021c-03de-4b53-8870-9370df147812.png)

Ubuntu Calendar:

![image](https://user-images.githubusercontent.com/7427347/202691524-6cf996aa-efbe-4c5b-8009-67c831841ea2.png)

MacOS with emoji that is not available:
<img width="284" alt="Screenshot 2022-11-18 at 12 33 53" src="https://user-images.githubusercontent.com/7427347/202696530-12283776-9ff1-4118-baf1-146ae2701e7b.png">

Nothing crashed, nothing failed to sync.

**Comment by joaquimlbacao-ist** [comment]
Hi there! I'd like to know if I can start working on this feature, since it has no assignees yet.

**Comment by miaulalala** [comment]
> Hi there! I'd like to know if I can start working on this feature, since it has no assignees yet.

Yes, that would be awesome! @jancborchardt can you weight in on the design?

**Comment by joaquimlbacao-ist** [comment]
Thank you for your response!
